src/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const secret = process.env.JWTSECRET;
+
+const verifyUser = (req, res, next) => {
+  const { token } = req.headers;
+  try {
```

Comment:
Please, go through your code and ensure that anytime you declare a variable or sets of variables, leave a line.
Example, leave a line after `const { token } = req.headers;`